assets/_scss/components/_accordions.scss

```diff
@@ -0,0 +1,46 @@
+#accordion-menu {
+  border-color: $color-grey-600;
+  border-style: solid;
+  border-width: 1px;
+  border-bottom-width: 0;
+  color: $color-grey-900;
+  margin: 0;
+  padding: 0;
+
+  li {
```

Comment:
OK thats fine, we can keep things like this until we see there being an actual problem in testing.

It would be good to make sure all devs are on the same page in how we are dealing with nested parts of components like this.